Meraki  MR18: opkg_install_cmd: Cannot install package nu801

### Describe the bug

Cant't build image for Meraki MR18 due to a missing package nu801, which is responsible  for  LED indicator 

```bash
Collected errors:
 * opkg_install_cmd: Cannot install package nu801.
 * opkg_install_cmd: Cannot install package nu801.
make[2]: *** [Makefile:189: package_install] Error 255
make[1]: *** [Makefile:144: _call_image] Error 2
make: *** [Makefile:262: image] Error 2
```

Latest available package --> releases/19.07.3/targets/ar71xx/tiny/

```bash
Package: kmod-leds-nu801
Version: 4.14.180-1
Depends: kernel (=4.14.180-1-114d5676c49eba1190f3a23a556ce343)
License: GPL-2.0
Section: kernel
Architecture: mips_24kc
Installed-Size: 105
Filename: kmod-leds-nu801_4.14.180-1_mips_24kc.ipk
Size: 845
SHA256sum: 7a45c9e2882cd0147d22016a94d35aee754867f55863dc643168c59fa704a919
Description:  Kernel module for the nu801 LED driver used on the Meraki MR18
```

### OpenWrt version

r23630-842932a63d

### OpenWrt release

23.05.2, 23.05.5

### OpenWrt target/subtarget

ath79/nand

### Device

Meraki MR18

### Image kind

Self-built image

### Steps to reproduce

make image PROFILE="meraki_mr18"

### Actual behaviour

Collected errors:
 * opkg_install_cmd: Cannot install package nu801.
 * opkg_install_cmd: Cannot install package nu801.
make[2]: *** [Makefile:189: package_install] Error 255
make[1]: *** [Makefile:144: _call_image] Error 2
make: *** [Makefile:262: image] Error 2

### Expected behaviour

_No response_

### Additional info

_No response_

### Diffconfig

_No response_

### Terms

- [X] I am reporting an issue for OpenWrt, not an unsupported fork.

--- a/package/system/gpio-cdev/nu801/Makefile
+++ b/package/system/gpio-cdev/nu801/Makefile
@@ -3,6 +3,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=nu801
+PKG_FLAGS:=nonshared
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git

